lib/iris/tests/integration/test_trajectory.py

```diff
@@ -233,7 +233,7 @@ class TestLazyData(tests.IrisTest):
     def test_hybrid_height(self):
         cube = istk.simple_4d_with_hybrid_height()
         # Put a biggus array on the cube so we can test deferred loading.
-        cube.data = da.from_array(cube.data, chunks=cube.data.shape)
+        cube.data = as_lazy_data(cube.data)
```

Comment:
Primarily this has been done for consistency, but there are also performance improvements to be had for setting the chunk size to the magic chunk size.